src/Core/src/Platform/Windows/RadioButtonExtensions.cs

```diff
@@ -47,6 +48,21 @@ public static void UpdateContent(this RadioButton nativeRadioButton, IRadioButto
 			else
 				nativeRadioButton.Content = $"{radioButton.Content}";
 		}
-			
+
+		public static void UpdateStrokeColor(this RadioButton nativeRadioButton, IRadioButton radioButton)
+		{
+			nativeRadioButton.BorderBrush = radioButton.StrokeColor != null ? radioButton.StrokeColor.ToNative() : (WBrush)UI.Xaml.Application.Current.Resources["ButtonBorderThemeBrush"];
+		}
+
+		public static void UpdateStrokeThickness(this RadioButton nativeRadioButton, IRadioButton radioButton)
+		{
+			nativeRadioButton.BorderThickness = radioButton.StrokeThickness == -1 ? WinUIHelpers.CreateThickness(3) : WinUIHelpers.CreateThickness(radioButton.StrokeThickness);
```

Comment:
```suggestion
			nativeRadioButton.BorderThickness = radioButton.StrokeThickness == <= 0 ? WinUIHelpers.CreateThickness(0) : WinUIHelpers.CreateThickness(radioButton.StrokeThickness);
```

Setting this to 3 by default was causing the border to always show up unless explicitly set to 0.